src/deprecated.jl

```diff
@@ -16,8 +16,8 @@ function by{T,N}(ta::TimeArray{T,N}, t::Int; period::Function=day)
     ta[rownums]
 end 
  
-function by{T,N}(ta::TimeArray{T,N}, t::ASCIIString; period::Function=day) 
-    warn("by(ta::TimeArray, t::ASCIIString; period::Function) is being deprecated.\nPlease use when(ta::TimeArray, period::Function, t::ASCIIString)")
+function by{T,N}(ta::TimeArray{T,N}, t::String; period::Function=day) 
+    warn("by(ta::TimeArray, t::String; period::Function) is being deprecated.\nPlease use when(ta::TimeArray, period::Function, t::String)")
```

Comment:
My pleasure. :) I can submit a PR for it later, if you'd prefer that to an issue.